### PR TITLE
Make Key.jsNative skiko-agnostic

### PR DESCRIPTION
**Pull request by Schahen**
This is the (rather small actually) part of work described in COMPOSE-1168-key-event

### CODE REVIEW

File: compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/Key.jsNative.kt [hunk 1]
```diff
@@ -28,345 +27,345 @@ import org.jetbrains.skiko.*
...
-        actual val Enter: Key = skikoKeyToKey(SkikoKey.KEY_ENTER)
+        actual val Enter = Key(13)
 
         /**
          * Backspace key.
          *
          * Deletes characters before the insertion point, unlike [Delete].
          */
-        actual val Backspace: Key = skikoKeyToKey(SkikoKey.KEY_BACKSPACE) // Key(KeyEvent.VK_BACK_SPACE)
+        actual val Backspace = Key(8) // Key(KeyEvent.VK_BACK_SPACE)
```
Comment (MatkovIvan) [inline compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/Key.jsNative.kt]:
```suggestion
        actual val Backspace = Key(8)
```

File: compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/Key.jsNative.kt [hunk 2]
```diff
@@ -28,345 +27,345 @@ import org.jetbrains.skiko.*
...
-        actual val Escape: Key = skikoKeyToKey(SkikoKey.KEY_ESCAPE)
+        actual val Escape = Key(27)
 
         /** Left Control modifier key. */
-        actual val CtrlLeft: Key = skikoKeyToKey(SkikoKey.KEY_LEFT_CONTROL)
+        actual val CtrlLeft = Key(17)
 
         /** Right Control modifier key. */
-        actual val CtrlRight: Key = skikoKeyToKey(SkikoKey.KEY_RIGHT_CONTROL)
+        actual val CtrlRight = Key(-2147483631) // 0x80000000.toInt() or 17
```
Comment (MatkovIvan) [inline compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/Key.jsNative.kt]:
```suggestion
        actual val CtrlRight = Key(0x80000000.toInt() or 17)
```

It's even better to Move `0x80000000` to named "mask" const

Comment (Schahen) [reply compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/Key.jsNative.kt]: @MatkovIvan what will benefit from the fact that this values are masked? Does it helps in certain scenarios? 
I'm not arguing - I just want to see a bigger picture, to understand what was initial intention behind this

Comment (MatkovIvan) [reply compose/ui/ui/src/jsNativeMain/kotlin/androidx/compose/ui/input/key/Key.jsNative.kt]: To get clear readable code like:
```kt
actual val CtrlLeft = Key(LEFT_MASK or 17)
actual val CtrlRight = Key(RIGHT_MASK or 17)
```

**Review by MatkovIvan — APPROVED** [review]

**Pull request merged**